utilities/sinsp-builder.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# This script is meant to be run from inside a collector-builder image. It will
+# create the driver for the current system, as well as Falco's sinsp-example
+# binary, which is useful for quickly testing driver changes.
+#
+# Run the collector-builder image with the following command:
+#   docker run --rm -it --entrypoint bash \
+#       --privileged \
+#       -v /var/run/docker.sock:/var/run/docker.sock \
+#       -v /sys:/sys:ro \
+#       -v /dev:/dev \
+#       -v /tmp:/tmp \
+#       quay.io/stackrox-io/collector-builder:cache
```

Comment:
It is a bit annoying to copy and paste this and remove the #s. Perhaps put this in a script and mention the script here. Perhaps create a new directory within the utilities directory called sinsp-builder, put this script and the docker run script there. You could also move a lot of this into a readme file.

I am also thinking that the utilities directory could be reorganized. That doesn't have to be done in this PR. I think the other scripts in the utilities directory could be in their own directories with readme files, instead of having a docs directory.

Reply:
> It is a bit annoying to copy and paste this and remove the #s. Perhaps put this in a script and mention the script here. Perhaps create a new directory within the utilities directory called sinsp-builder, put this script and the docker run script there. You could also move a lot of this into a readme file.

I really don't want to move this to a script, depending on whether you are running on an immutable system or not the command is different and we already have a way to run the builder image using `make start-dev` for local purposes. This script is meant to slightly simplify the process of building this binary and the driver when testing in remote systems were you might not have the repo cloned, which means you will need to put some elbow grease to get it working anyways, it's just not as much elbow grease as it would be if you need to do all of this on your own.

I also don't mean for this to become an entire framework for us to run the `sinsp-example` binary, so splitting it into its own directory, adding a readme, etc. it all feels like overkill to me.

> I am also thinking that the utilities directory could be reorganized. That doesn't have to be done in this PR. I think the other scripts in the utilities directory could be in their own directories with readme files, instead of having a docs directory.

We can discuss this with the broader team, I agree the state of the utilities directory is not great, but we also kinda use it as a place to drop scripts that might be useful on their own. Keeping separate README files for each script we might come up with seems like a lot of work to put together and maintain. I wrote every possible nuisance I could come up with into the script because that way people don't need to go looking for it on a separate file and, if someone makes a change to the script, it will be pretty much obvious that you'll need to make some adjustments to the "docs" that is that comment.